ring_buffer/ring_buffer.py

```diff
@@ -5,7 +5,29 @@ def __init__(self, capacity):
     self.storage = [None]*capacity
 
   def append(self, item):
-    pass
+    if self.current >= self.capacity:
```

Comment:
I would reverse the order of the if statement and line 10 and 11. I think this would be cutting it one short, no? Other than that, you are spot on.